packages/grafana-data/src/panel/getPanelOptionsWithDefaults.ts

```diff
@@ -56,8 +56,8 @@ function applyFieldConfigDefaults(existingFieldConfig: FieldConfigSource, plugin
   const result: FieldConfigSource = {
     defaults: mergeWith(
       {},
-      pluginDefaults.defaults,
       existingFieldConfig ? existingFieldConfig.defaults : {},
+      pluginDefaults.defaults,
       (objValue, srcValue) => {
         if (isArray(srcValue)) {
```

Comment:
@jackw not sure this works either as the merge function will always take the source array if it's an array. So with existing config added before the defaults means an array with values will always override the the current config array. 

Or am I missing something?